### ISSUE TITLE
frozen_string_literal

### ISSUE THREAD

**Issue by igor-alexandrov** [dry-rb/dry-validation#458]
@solnic @timriley I think we should add `# frozen_string_literal: true` comment, since we do not modify strings and should enforce this.

**Comment by timriley** [comment]
I agree, @igor-alexandrov.

I might leave it for @solnic to do this before release, though, since it's probably best done directly rather than via a PR, which might get out of date.

@solnic here's how I did it in dry-view, recently:

```sh
comm -23 \
<(git ls-files|sort) \
<(git grep -l 'frozen_string_literal'|sort) \
| grep -E '\.rb$' \
| xargs -n1 sed -i "" -e '1s/^/# frozen_string_literal: true\
\
/'
```

That is the command required for the macOS version of `sed`, which (where newlines must be literal as well as backslash-escaped). The line breaks here must be copied as is.

Hopefully this saves you the hour or so I spent trying to get it to work on my machine :)

**Comment by igor-alexandrov** [comment]
Guys, rubocop can do this out of the box:
```bash
rubocop --only Style/FrozenStringLiteralComment -a
```

**Comment by solnic** [comment]
TIL `--only` option in rubocop! Thanks @igor-alexandrov 😄 I'll do it now.

@timriley nice shell skills mate 😄

**Comment by solnic** [comment]
Fixed via 702a99757fc925b0700a9dd97bea0b55b794bab7 (I used wrong issue id in the commit message ;()
